### PR TITLE
feat: new flag to exclude columns from schema validation

### LINKED ISSUE #382: Exclude certain columns from schema validation
There should be a method to exclude column in schema validation

### PR DESCRIPTION
**Pull request by helensilva14**
Closes issue #382

### CODE REVIEW

**Comment by helensilva14** [comment]
New feature depends on PR #512 merging to fix the execution of the schema validation

/gcbrun

File: data_validation/cli_tools.py [hunk 1]
```diff
@@ -551,8 +551,13 @@ def _configure_column_parser(column_parser):
 
 
 def _configure_schema_parser(schema_parser):
-    """Configure arguments to run column level validations."""
+    """Configure arguments to run schema level validations."""
     _add_common_arguments(schema_parser)
+    schema_parser.add_argument(
+        "--exclusion-columns",
```
Comment (nehanene15) [inline data_validation/cli_tools.py:557]: Let's document this in our README - https://github.com/GoogleCloudPlatform/professional-services-data-validator#schema-validations

**Review by nehanene15 — DISMISSED** [review]
Only thing missing is to document the new flag in the README. Otherwise, LGTM!

**Review by nehanene15 — APPROVED** [review]
LGTM!

**Pull request merged**